homeassistant/components/light/hue.py

```diff
@@ -47,7 +47,7 @@
 
 PHUE_CONFIG_FILE = 'phue.conf'
 
-SUPPORT_HUE_ON_OFF = (SUPPORT_FLASH | SUPPORT_TRANSITION | SUPPORT_FLASH)
+SUPPORT_HUE_ON_OFF = (SUPPORT_FLASH | SUPPORT_TRANSITION)
```

Comment:
`SUPPORT_FLASH` needs to be reintroduced somewhere in the chain here. Presumably at `SUPPORT_HUE_DIMMABLE`, but I'm not sure.